MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs

```diff
@@ -58,18 +123,127 @@ public bool ValidateVersion(string encoderAppPath, bool logOutput)
                 return false;
             }
 
-            output = " " + output + " ";
+            // The minimum FFmpeg version required to run jellyfin successfully
+            FFmpegVersion required = new FFmpegVersion("4.0");
```

Comment:
The minimum recommendation..

Reply:
Well, it's actually is maximum recommendation as well :) And I see no point in your validation code that checks for `ffmpeg < 4.1`. We currently support _only_ `4.0.x`, anything else breaks at different places.

Reply:
Are you suggesting we need something more like the following, where the range might grow going forward:
```
FFmpegVersion minRequired = new FFmpegVersion("4.0");
FFmpegVersion maxRequired = new FFmpegVersion("4.0");
```